src/attr.rs

```diff
@@ -6,12 +6,16 @@ use std::iter;
 use proc_macro2::TokenStream;
 
 #[cfg(feature = "parsing")]
+#[cfg_attr(syn_enable_doc_cfg, doc(cfg(feature = "parsing")))]
```

Comment:
These are private `use` statements which aren't re-exported, so no need to attach `doc(cfg(` attributes to them.

Reply:
I think the sed-regex (replace all) did more harm than good. I remove the whole commit and add the necessary cfgs by hand